### DISCUSSION TITLE
Error occurs when compile:     /edk2-master/BaseTools$ make -C Source/C

### DISCUSSION THREAD

**Discussion by MingchengLi** [tianocore/edk2#3415] [Q&A]
Hi guys, when I do the compile, it goes errors. Could you guys help me with that?
here is the error:
![image](https://user-images.githubusercontent.com/84122187/192417478-fb4db830-830b-472f-b886-9498e8193923.png)

what I've done was like compile the file by "make":
![image](https://user-images.githubusercontent.com/84122187/192417590-5e98cdcf-1be0-4f51-9a3f-e7ecdc668f6a.png)

IDK where is the problem, i just want to check if it can be compiled

**Comment by lersi** [comment]
what version of ubuntu are you using?
from my experience, older ubuntu versions (<2022) don't work well with the new releases.
try to use older EDK2 release, from 2020.

**Comment by Elemeants** [comment]
Hi!

Did you do a `git submodule init && git submodule update --recursive`?
The header file already exists on that path?
